.github/workflows/build.yml

```diff
@@ -196,6 +196,7 @@ jobs:
         - "pyb_nano_v2"
         - "pybadge"
         - "pybadge_airlift"
+        - "pycubed"
```

Comment:
You just need move this so it's in alphabetical order. That should fix the checks failure.